[2022-08-07 19:57:03,448] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: fraud_generation.generate_transaction scheduled__2022-08-07T15:57:00.868812+00:00 [queued]>
[2022-08-07 19:57:03,458] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: fraud_generation.generate_transaction scheduled__2022-08-07T15:57:00.868812+00:00 [queued]>
[2022-08-07 19:57:03,458] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-07 19:57:03,458] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-08-07 19:57:03,458] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-07 19:57:03,472] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): generate_transaction> on 2022-08-07 15:57:00.868812+00:00
[2022-08-07 19:57:03,479] {standard_task_runner.py:52} INFO - Started process 170 to run task
[2022-08-07 19:57:03,484] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'fraud_generation', 'generate_transaction', 'scheduled__2022-08-07T15:57:00.868812+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/fraud_generation_dag.py', '--cfg-path', '/tmp/tmp32sujdfy', '--error-file', '/tmp/tmpi1c60bdc']
[2022-08-07 19:57:03,485] {standard_task_runner.py:80} INFO - Job 3: Subtask generate_transaction
[2022-08-07 19:57:03,580] {task_command.py:370} INFO - Running <TaskInstance: fraud_generation.generate_transaction scheduled__2022-08-07T15:57:00.868812+00:00 [running]> on host 42f3319e71a2
[2022-08-07 19:57:03,677] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fraud_generation
AIRFLOW_CTX_TASK_ID=generate_transaction
AIRFLOW_CTX_EXECUTION_DATE=2022-08-07T15:57:00.868812+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-07T15:57:00.868812+00:00
[2022-08-07 19:57:03,679] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-07 19:57:03,680] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python3 /opt/***/dags/script_generation.py 13811']
[2022-08-07 19:57:03,694] {subprocess.py:85} INFO - Output:
[2022-08-07 19:57:05,500] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-07 19:57:05,534] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fraud_generation, task_id=generate_transaction, execution_date=20220807T155700, start_date=20220807T195703, end_date=20220807T195705
[2022-08-07 19:57:05,590] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-07 19:57:05,624] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
